keps/prod-readiness/sig-scheduling/3101.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3101
+beta:
+  approver: "@ahg-g"
```

Comment:
@ahg-g not sure who should I add as prr approver?

Reply:
leave it for now, will add it later

Reply:
I'm happy to take it.

Reply:
Great, thanks wojtek!

Reply:
Thanks @wojtek-t!